tests/python/test_element_wise.py

```diff
@@ -86,6 +86,61 @@ def func():
             assert c[None][i, j] == approx(expected)
 
 
+@ti.host_arch_only
+def _test_matrix_element_wise_inplace_binary(dtype,
```

Comment:
1. `inplace` -> `writeback`
2. is the `_` prefix intentional?

Reply:
2. Yes, it's a helper function, or I'll use `pytest.parameterize` to fill up the arguments.